catalog/core/catalog-core-localstorageprovider/src/main/java/org/codice/ddf/catalog/content/impl/FileSystemStorageProvider.java

```diff
@@ -142,6 +142,10 @@ public CreateStorageResponse create(CreateStorageRequest createRequest)
     public ReadStorageResponse read(ReadStorageRequest readRequest) throws StorageException {
         LOGGER.trace("ENTERING: read");
 
+        if (readRequest.getResourceUri() == null) {
+            return new ReadStorageResponseImpl(readRequest);
```

Comment:
This `ReadStorageResponseImpl` object will contain a `null` item, which means that its `getContentItem()` will return `null` and could break existing code.

Fortunately, the interface that returns this is marked experimental so this isn't an API change and as long as we know any of our code isn't going to break because of this change, it should be ok to simply add `@Nullable` on the `getContentItem()` return type.